source/operations/install-deploy-manage/deploy-operator-helm.rst

```diff
@@ -0,0 +1,316 @@
+.. _minio-k8s-deploy-operator-helm:
+
+=========================
+Deploy Operator With Helm
+=========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+
+Overview
+--------
+
+Helm is a tool for automating the deployment of applications to Kubernetes clusters.
+A `Helm chart <https://helm.sh/docs/topics/charts/>`__ is a set of YAML files, templates, and other files that define the deployment details.
+The following procedure uses a Helm Chart to install the :ref:`MinIO Kubernetes Operator <minio-operator-installation>` to a Kubernetes cluster.
+
+
+Prerequesites
+-------------
+
+To install the Operator with Helm you will need the following:
+
+* An existing cluster using Kubernetes 1.19.0 or later (1.21.0 or later recommended).
+* The ``kubectl`` CLI tool on your local host, the same version as the cluster.
+* `Helm <https://helm.sh/docs/intro/install/>`__ version xx or greater.
+* `yq <https://github.com/mikefarah/yq/#install>`__ version xx or greater.
+* Access to run ``kubectl`` commands on the cluster from your local host.
+
+For more about Operator installation requirements, including TLS certificates, see the :ref:`Operator deployment prerequisites <minio-operator-prerequisites>`.
+
+This procedure assumes familiarity the with referenced Kubernetes concepts and utilities.
+While this documentation may provide guidance for configuring or deploying Kubernetes-related resources on a best-effort basis, it is not a replacement for the official :kube-docs:`Kuber\
+netes Documentation <>`.
+
+
+Procedure
+---------
+
+
+Install Operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Download the Helm charts
```

Comment:
There's a lot of nested stuff here, looking for ideas for making it easier to navigate. I'm not fond of nested lists, but is that better than several more levels of headers?